packages/atlas/src/hooks/useReactionTransactions.ts

```diff
@@ -0,0 +1,79 @@
+import { QueryResult } from '@apollo/client'
+import { useState } from 'react'
+
+import { ReactionId } from '@/components/_comments/ReactionChip'
+import { VideoReaction } from '@/joystream-lib'
+import { useJoystream } from '@/providers/joystream'
+import { useTransaction } from '@/providers/transactionManager'
+import { useUser } from '@/providers/user'
+import { ConsoleLogger } from '@/utils/logs'
+
+import { useDisplaySignInDialog } from './useDisplaySignInDialog'
+
+export const useReactionTransactions = (refetch: QueryResult['refetch']) => {
```

Comment:
Is there a good reason to provide `refetch` via an argument versus doing it inside this hook, similar to what we do for nft transactions? One issue would be if you wanted to react to both video and comments from a single component, you would be able to only provide a single `refetch`